src/pudl/glue/epacamd_eia.py

```diff
@@ -138,5 +143,400 @@ def transform(
             on=["plant_id_eia", "boiler_id"],
             how="inner",
         )
-
+    # TODO: Add mannual crosswalk
     return {"epacamd_eia": crosswalk_clean}
+
+
+##################
+# Add Sub-Plant ID
+##################
+
+
+# @asset
+def identify_subplants_epacamd_eia(
```

Comment:
If `identify_subplants_epacamd_eia` is a table we'd like to persist in the database I'd recommend turning it into an asset. If it just a cleaning function that augments to the `epacamd_eia` asset, I'd keep it as a python function.

Where is `identify_subplants_epacamd_eia` being used?

Reply:
its not being used rn! I was planning on figuring out the dag of it all before slotting it in.  I was thinking we could have a clean_epacamd_eia asset (result of `pudl.glue.epacamd_eia.transform`) which feeds into this second stage of `epacamd_eia` asset. kind of in line with the clean_{table}_eia860 -> {table}_eia860 model? but also we could just append=sin this to the cleaning steps within `pudl.glue.epacamd_eia.transform`.

Reply:
hm! no i think we need to make this in two steps. bc CEMS relies on this epacamd crosswalk and now this sub-plant-id relies on both the cleaned crosswalk and CEMS.

Reply:
Correct! We can't have any circular dependencies. I think we just need to create convert `identify_subplants_epacamd_eia` to be a dagster asset that depends on `hourly_emissions_epacems`, `epacamd_eia`, and `generators_eia860`. 

To depend on `hourly_emissions_epacems` you'll need to specify [the io manager](https://docs.dagster.io/concepts/io-management/io-managers#asset-input-io-managers):

```python
@asset(
    ins={
        "first_asset": AssetIn(input_manager_key="epacems_io_manager"),
        # I'm not sure if we'll have to specify the io managers for the other upstream dependencies
    }
)
def identify_subplants_epacamd_eia(
    epacamd_eia: pd.DataFrame,
    generators_eia860: pd.DataFrame,
    hourly_emissions_epacems: dd.DataFrame # note this will be a dask dataframe
) -> pd.DataFrame:
  ...
```